src/components/table/table-header/table-header.component.js

```diff
@@ -282,16 +178,73 @@ class TableHeader extends React.Component {
     } else {
       contents = this.props.children;
     }
-
     return (
-      <th
-        { ...this.tableHeaderProps } { ...this.componentTags(this.props) }
-        { ...this.ariaAttributes() }
-      >
-        { contents }
-      </th>
+      React.createElement(
+        this.styledComponent(),
+        {
+          ...this.componentTags(this.props),
+          ...this.tableHeaderProps,
+          ...this.ariaAttributes(),
+          align: this.props.align,
+          sortable: this.props.sortable
+        },
+        contents
+      )
     );
   }
 }
 
+TableHeader.propTypes = {
+
+  /**
```

Comment:
I think the preference in the team is for these propType comments to be on a single line.